Sources/Clappr/Classes/Base/Container.swift

```diff
@@ -63,6 +83,23 @@ open class Container: UIObject {
         }
     }
 
+    @objc open func reload() {
```

Comment:
What is the reason for this method being public and present in our published API?

Reply:
Fixed